openrtb_ext/floors.go

```diff
@@ -0,0 +1,45 @@
+package openrtb_ext
+
+type PriceFloorSchema struct {
```

Comment:
Please add comments for where within the openrtb request these apply to.

Reply:
Added comment